frontend/src/__mocks__/third_party/mlmd/generated/google/protobuf/any.ts

```diff
@@ -0,0 +1,240 @@
+// Code generated by protoc-gen-ts_proto. DO NOT EDIT.
```

Comment:
Could we add a eslint rule to prevent people from importing this outside of this __mock__ directory?

something like
```json
{
  "rules": {
    "no-restricted-imports": [
      "error",
      {
        "patterns": [
          {
            "group": ["./src/__mocks__/third_party/mlmd/*"],
            "message": "Importing from './src/__mocks__/third_party/mlmd/' is restricted to '__mocks__/mlmd/'."
          }
        ]
      }
    ]
  },
  "overrides": [
    {
      "files": ["./src/__mocks__/mlmd/**"],
      "rules": {
        "no-restricted-imports": "off"
      }
    }
  ]
}
```